webpack/webpack.config.base.js

```diff
@@ -208,6 +204,12 @@ const config = {
     new CopyWebpackPlugin({
       patterns: [{ from: 'node_modules/webextension-polyfill/dist/browser-polyfill.js' }],
     }),
+
+    new Dotenv({
+      allowEmptyValues: true,
+      systemvars: IS_PROD,
```

Comment:
Can't we have this always as `true`? Locally we'll have no system vars set, and so it'll use the .env vars anyway? 🤔